### DISCUSSION TITLE
WebDav中显示的文件夹可以隐藏吗？

### DISCUSSION THREAD

**Discussion by iShao** [AlistGo/alist#4150] [Q&A]
在Web端中通过全局添加了如下把`$RECYCLE.BIN`和`System Volume Information`两个文件夹给隐藏掉了
```
/\/\$RECYCLE.BIN/i
/\/System Volume Information/i
```

但是在webdav中还是会显示出来， 在元信息中尝试添加这两个也没有生效

**Comment by XinSSS** [comment]
尝试了下，应该可以隐藏， 不过我没用上面两个表达式
我用的是这个
```
\$RECYCLE.BIN
System Volume Information
```

另外还能看到这两个文件夹的话， 尝试用非admin账号登录看看，admin是可以看到隐藏文件夹的

**Reply by iShao** [reply]
搞定了， 多谢。。但是好像发现了一个Bug
我在元信息中另外添加了一条记录隐藏一个路径下的子文件夹。
子文件夹确实被隐藏了， 但是这个目录下的System Volume Information和$RECYCLE.BIN又显示出来了。。。。。。

**Comment by anwen-anyi** [comment]
如果你登录的是admin隐藏是无效的
但是你在webdav登录的是没有管理权限的webdav用户是看不到隐藏的
